index.html

```diff
@@ -20,6 +20,16 @@
 
     </head>
     <body >
+	    
+	    
+	    <div class="lang_switcher">
+        <center>
+            <button type="submit" name="lang" value="Irish" onclick="Irish()">Gaeilge</button>
+            <button type="submit" name="lang" value="English" onclick="English()">English</button>
+        </center>
+    </div>
+	    
+	    
         <div class="container">
             <div class="imagebg"></div>
             <div class="row " style="margin-top: 50px">
```

Comment:
make sure that your buttons work for that you should add javascript to change the language on a webpage.

Reply:
make sure that your buttons work for that you should add javascript to change the language on a webpage.